src/js/themes/base.js

```diff
@@ -1036,6 +1040,67 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // colors: [] || colors: ['graph-0', 'graph-1', 'graph-2', 'graph-3'],
       // extend: undefined,
     },
+    notification: {
+      container: {
+        pad: { horizontal: 'small', vertical: 'xsmall' },
```

Comment:
If these containers accept any Box props (as indicated in `base.d.ts` we have indicated so by including a comment like here: https://github.com/grommet/grommet/blob/89e8c5fea22324804c806d5871d5237a8cd99573/src/js/themes/base.js#L735

This would apply to the other containers as well.

```suggestion
      container: {
        // any Box props
        pad: { horizontal: 'small', vertical: 'xsmall' },
```